colcon_cmake/test_result/ctest.py

```diff
@@ -96,6 +96,25 @@ def get_test_results(
                     result.skipped_count += 1
                 elif status != 'passed':
                     result.error_count += 1
+
+                if collect_details and status == 'failed':
+                    lines = []
+                    lines.append('('+child.findtext('Name')+')')
```

Comment:
Please avoid the surrounding parenthesis since they don't seem to match any other verbose test result reporting.

Nitpick: merge this line with the line above.

Reply:
done